client/src/components/Portfolio.tsx

```diff
@@ -0,0 +1,69 @@
+import Navbar from "./Navbar"
+import styled from "styled-components"
+import { ReactNode } from "react"
+import {Button, Container, makeStyles, Typography} from "@material-ui/core"
+
+const useStyles = makeStyles(theme => ({
+    container: {
+        marginLeft: theme.spacing(10),
+        marginRight: theme.spacing(10),
+        marginTop: theme.spacing(0),
+        display: "flex",
+        flexDirection: "column"
+    },
+    pricesContainer: {
+       display: "flex",
+       flexDirection: "row"
+    },
+    changeComponent: {
+        background: "#00FF00",
+        color: "white",
+        fontWeight: "bold",
+        marginLeft: theme.spacing(20)
+    },
+    changeValueComponent: {
+        color: "#00FF00"
+    }
+}))
+
+function numberWithCommas(valueStr: string) {
+    return valueStr.replace(/\B(?=(\d{3})+(?!\d))/g, ",");
+}
+
+function calculatePortfolioValue(cryptos: any[]): number {
+
+    let value = 0
+    cryptos.forEach(crypto => {
+        value += crypto.price
+    })
+
+    return value
+}
+
+const Portfolio = (props: any) => {
+
+    const classes = useStyles()
+    return (
+        <div className={classes.container}>
```

Comment:
Is it possible to replace this with `<Container component="div">...`? I'm not sure if it's recommended to mix material UI items with regular HTML.

Reply:
Also for `<div>`s below.